Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc

```diff
@@ -526,7 +537,15 @@ void TrackerGeometryCompare::compareSurfaceDeformations(TTree* refTree, TTree* c
             if (TMath::Abs(surfDeform_[npar]) > (m_rangeHigh_ - m_rangeLow_) / (10. * m_nBins_))
               m_h1_[histname2.str()]->Fill(surfDeform_[npar]);
           }
-          (m_vtkmap_.at(npar)).fill_current_val(inputRawid1, surfDeform_[npar]);
+          if (phase_ < SiPixelPI::phase::two) {
+            if (phase_ == SiPixelPI::phase::zero) {
+              (m_vtkmap_.at(npar)).fill_current_val(inputRawid1, surfDeform_[npar]);
+            } else {
+              if (!(inputSubdetid1 == 1 && inputSubdetid2 == 1) && !(inputSubdetid1 == 2 && inputSubdetid2 == 2)) {
```

Comment:
leave a comment that the "old-style" Tracker Map does not support Phase-1 pixels ?